### ISSUE TITLE
zig build advices the use of deprecated commands

### ISSUE THREAD

**Issue by bortzmeyer** [ziglang/zig#18101]
### Zig Version

0.12.0-dev.1685+994e19164

### Steps to Reproduce and Observed Output

```
% zig build
info: Initialize a 'build.zig' template file with `zig init-lib` or `zig init-exe`,
or see `zig --help` for more options.
error: No 'build.zig' file found, in the current directory or any parent directories.

% zig init-lib
info: Usage: zig [command] [options]

Commands:

  build            Build project from build.zig
  fetch            Copy a package into global cache and print its hash
  init             Initialize a Zig package in the current directory
...
```

### Expected Output

`init-lib`  and `init-exe` aparently no longer exist, there is only one `init`.

**Comment by jedisct1** [comment]
Good catch!

#18103 should fix it.

**Comment by sommerper** [comment]
I'm trying to learn zig and following this guide does not work
https://ziglang.org/learn/getting-started/
It mentions `init-exe`.

**Comment by asnaseer** [comment]
> I'm trying to learn zig and following this guide does not work https://ziglang.org/learn/getting-started/ It mentions `init-exe`.

Replace `init-exe` with `init` - I believe this documentation page also needs updating.

**Comment by andrewrk** [comment]
For the CLI, this was solved in 0c0b69891ad0461a7cd14b09a68def350fbf128e. I fixed the website just now.
